### ISSUE TITLE
Inline calculation of IVT for AR detection

### ISSUE THREAD

**Issue by adamrher** [ESCOMP/CAM#536]
Inline calculations of IVT,uIVT,vIVT (vertically integrated moisture transport) in cam_diagnostics.F90 has been used by AR researchers (@shieldsca) as SourceMods for some time now ... so I'm opening this issue to put this additional diagnostic on the trunk. @andrewgettelman Christine said you planned on getting this in at some point, but didn't see an issue. Happy to include this in the upcoming CLUBB PR since that will have additional diagnostics in it as well (i.e.,  https://github.com/ESCOMP/CAM/issues/497).

**Comment by andrewgettelman** [comment]
I have no plans to work on this or put it in.  But we have code in there (Xingying Huang has it I believe), so someone could do a PR perhaps. I can see if Xingying wants to try.

**Comment by adamrher** [comment]
I'll probably merge it in with the new clubb externals PR, as long the SE's are cool w/ that. I have the code.

**Comment by andrewgettelman** [comment]
If you have the code, that's a great plan (IMHO).

**Comment by cacraigucar** [comment]
It sounds fine to me for @adamrher's PR to combine this issue along with the changes required in #497.  It sounds like this issue will only be making mods to cam_diagnostics.F90, correct?

**Comment by adamrher** [comment]
> It sounds like this issue will only be making mods to cam_diagnostics.F90, correct?

yes.

**Comment by xhuang-ncar** [comment]
@adamrher Hi Adam, I was talking with Andrew (@andrewgettelman) yesterday to add these IVT diagnostic codes from Christine to cam_diagnostics.F90. I just realized that we already have this issue open. Do you still plan to do this merge? I can help with this too if you'd like. Just let me know. Thanks. -Xingying

**Comment by adamrher** [comment]
yep, this is about to come in in my PR https://github.com/ESCOMP/CAM/pull/598. I would close the old issue with a note sayings its a duplicate.

**Comment by xhuang-ncar** [comment]
That's nice to know. Thanks.
